source/platforms/git/localGetHeadSHA.ts

```diff
@@ -0,0 +1,20 @@
+import { debug } from "../../debug"
+import { exec } from "child_process"
+
+const d = debug("localGetHeadSHA")
+
+export const localGetHeadSHA = () =>
+  new Promise<string>(done => {
+    const call = `git rev-parse HEAD"`
```

Comment:
```suggestion
    const call = `git rev-parse HEAD`
```

This typo looks like it might be the first level of why the PR was red 🟥